src/app/api/login_service/login.service.ts

```diff
@@ -1,41 +1,76 @@
 import { Injectable } from '@angular/core';
 
-import { UserService } from '../user_service/user.service';
+import { UserClass } from '../Classes/user';
+import { WeatherClass } from '../Classes/weather';
+import { DashboardService } from '../dashboard_service/dashboard.service';
 
 @Injectable()
 export class LoginService {
```

Comment:
Login service seems to be doing a bit more than just logging in.

- [ ] Rename to UserService or UserAuthenticationService